15-draft.json

```diff
@@ -18,6 +18,15 @@
       "description": "The name of your space",
       "type": "string"
     },
+    "type": {
+      "description": "The type of your space. Clients MUST treat a missing <code>type</code> field as if it were set to <code>space</code>. <code>space</code>: A hacker- or makerspace. For endpoints with <code>\"type\": \"space\"</code>, the <code>location</code> is mandatory. <code>hub</code>: An association of multiple people or spaces, which does not have a physical location itself.  Examples of hubs could be umbrella organizations or groups without a single hacker- or makerspace.",
+      "type": "string",
+      "default": "space",
+      "enum": [
+        "space",
+        "hub"
```

Comment:
I'm not totally convinced about the term "hub", but don't really have a better suggestion right now 😕

Reply:
Resolving this, since we decided to remove the type anyways.